<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>540</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>261</width>
     <height>455</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="formLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="cityLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>City</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="cityTextEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="addressLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="addressTextEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sizeLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="sizeTextEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="priceLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>price</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="priceTextEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="rentPriceLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Rent price</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="rentPriceTextEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>(Type 0 if this house is not for rent)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="bedroomCountLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Bedroom Count</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="bedroomCountTextEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="furnishedLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Furnished</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="furnishedTextEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>(True or False)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="otherLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Other</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="otherTextEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>(etc.)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addHouseButton">
      <property name="text">
       <string>Add House</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
